### PR TITLE
Release/6.0.0: Linking to Documentation instead of wiki

### PR DESCRIPTION
**Pull request by leonardoInf**
The wiki has less documentation than the gh-pages. So let people know that there is also https://chroma-sdk.github.io/Colore/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,8 +14,7 @@ A powerful and elegant C# library for Razer Chroma's SDK
 Getting started
 ---------------
 
-If you are a new developer and are looking for a helpful guide on how to get started, head on over to
-[the wiki page][getting-started] which describes getting Colore installed and running some example code.
+If you are a new developer and are looking for a helpful guide on how to get started, head on over to the [Documentation](https://chroma-sdk.github.io/Colore/docs/articles/getting-started.html) which describes getting Colore installed and running some example code.
```
Comment (Sharparam) [inline README.md]: Nitpick: "Documentation" should be lowercase here as it's a regular word and lines should strive to be at most 120 characters in length.

Comment (leonardoInf) [reply README.md]: ok, but you approve the idea?

Comment (Sharparam) [reply README.md]: Yeah I think it's a good idea to link to the docs version, at least for now as it is where effort should be put into documenting the library.

**Comment by leonardoInf** [comment]
Thx, can you pull it without Travis?

**Comment by Sharparam** [comment]
Yeah this has no code changes anyway. I'll fix up the CI tools so they work with external pull requests for next time.

**Pull request merged**